widget_driver/example/build.yaml

```diff
@@ -0,0 +1,11 @@
+targets:
+  $default:
+    builders:
+      widget_driver_generator:
+        options:
+          defaultTestValues:
+            "bool": "true"
+            "Color": "Colors.yellow"
+            "int": "123"
+            "Coffee": "const Coffee(\n    name: 'Coffee',\n    description: 'Some desc',\n    imageUrl: 'http://www.exampleImage.com/image',\n  )"
+            "String": "'Hello World'"
```

Comment:
This would be the way you submit default values. Note that you have to specify how they will be constructed, not something like a json version of the class. Best example for this is the Coffee